docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.1'
```

Comment:
Let remove this docker compose from this repo
it has Expetitle's names
we should keep this repo separate from Expetitle's related code.

We can have this docker-compose in exp-kratos service

Reply:
we dont need this docker-compose file, this was only for building kratos directly from code, so I could write the code easily and check quick, otherwise for sure I need to use docker compose from exp-kratos